components/bam.iobio/package.json

```
@@ -0,0 +1,31 @@
+{
+	"name": "@overture-stack/bam-visualizations",
+	"version": "0.1.0",
+	"private": true,
+	"type": "module",
+	"dependencies": {
+		"@emotion/react": "^11.11.4",
+		"bam.iobio": "file:../../../bam.iobio.io.poc",
```

I believe for these "local" dependencies, you may be able to do something like this https://github.com/overture-stack/lectern/blob/develop/apps/server/package.json#L42
```suggestion
		"bam.iobio": "workspace:^",
```

---

Good tool to be aware of but I don't think it fits this case.
`workspace` is being used to refer to files still within the project; I'm using `file` to refer to a directory outside of this project.
Jon pointed out we shouldn't commit `file:/../` either b/c it won't transfer to installs on any other cpu. This is also going to be replaced by `@iobio-charts`. So I'll just remove it for now and replace in the next ticket.
cc: @joneubank